Directory.Build.targets

```diff
@@ -9,11 +10,93 @@
     <PublishTrimmed Condition=" '$(TargetFramework)' == 'net6.0-ios' or '$(TargetFramework)' == 'net6.0-maccatalyst' ">false</PublishTrimmed>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(SingleProject)' == 'true'">
+    <!-- Required - Enable Launch Profiles for .NET 6 iOS/Android -->
+    <_KeepLaunchProfiles>true</_KeepLaunchProfiles>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(SingleProject)' == 'true'">
+    <!-- Required - Setup Single Project features -->
+    <ProjectCapability Include="MauiSingleProject" />
+
+    <!-- Optional - Enables a list of TFM's and device categories in the debug menu -->
+    <!-- This allows easily toggling of debug target TFM by selecting the platform -->
+    <!-- If removed, Top level debug targets show as a list of devices for the selected TFM -->
+    <ProjectCapability Include="XamarinStaticLaunchProfiles" />
+  </ItemGroup>
+
+  <!-- HACK: for windows -->
   <Target Name="Net6WinUIPackagingWorkaround" BeforeTargets="GetPackagingOutputs" Condition="$(TargetFramework.Contains('-windows')) == true AND '$(DisableEmbeddedXbf)' != 'true' ">
     <PropertyGroup>
       <TargetPlatformIdentifierAdjusted>UAP</TargetPlatformIdentifierAdjusted>
       <_SupportEmbedFileResources>true</_SupportEmbedFileResources>
       <_EmbedFileResfilePath>$(IntermediateOutputPath)$(AppxSubfolderWithFilesToBeEmbedded)\embed.resfiles</_EmbedFileResfilePath>
     </PropertyGroup>
   </Target>
+
+  <!-- HACK: Avoid the bad tasks for multitargeting -->
+  <Target Name="ValidateWinUIPlatform" />
+  <Target Name="BinPlaceBootstrapDll" />
```

Comment:
This is due to WinUI packages not supporting AnyCPU and they copy files based on "native" platforms - which should be done automatically with the RIDs.